docs/v3/source/includes/concepts/_metadata.md.erb

```diff
@@ -30,7 +30,7 @@ Example Resource with Annotations
 
 Annotations are user-specified key-value pairs that are attached to [API resources](#api-resource). They do not affect the operation of Cloud Foundry. Annotations cannot be used in [filters](#filters).
 
-Annotations set to organizations, spaces, service instances and apps resources are send to the service broker during creation of service instances or service credential bindings creations.
+Annotations with prefix set to organizations, spaces, service instances and apps resources are send to the service broker during creation of service instances, service credential bindings or service route bindings creations.
```

Comment:
The sentence has lost some clarity. How about:

When a service instance is being created, the service broker is sent the annotations for the service instance, and the space and organization in which the service instance reside. When a service binding is being created, the service broker is sent annotations for the service binding, any associated app, and the space and organization in which the app resides. Only annotations with a prefix are sent.

Reply:
Updated it to:

```
When a service instance is being created, the service broker is sent the annotations of the service instance, and the space and organization in which the service instance reside.
When a service instance is being updated, the service broker is sent the annotations of the space and organization in which the service instance reside.
When a service binding is being created, the service broker is sent annotations of any associated app, and the space and organization in which the binding resides.
Only annotations with a prefix are sent.
```